shared/js/ui/base/localize.es6.js

```diff
@@ -0,0 +1,37 @@
+import i18next from 'i18next'
+import ICU from 'i18next-icu'
+import config from '../../../data/constants.js'
+
+const supportedLocales = config.supportedLocales
+
+function buildLocalesList () {
+    const resources = {}
+    // hard coding for now till we figure out dynamic bundling
+    resources.en = {
+        shared: require('../../../locales/en/shared.json'),
+        options: require('../../../locales/en/options.json')
+    }
+    return resources
+}
+
+function getLocale (supportedLocales) {
+    // truncate locale by dropping country suffix
+    let browserLocale = chrome.i18n ? chrome.i18n.getUILanguage() : navigator.language
+    browserLocale = browserLocale.split('-')[0]
+
+    return supportedLocales.includes[browserLocale] || 'en' // probably redundant due to fallbackLng in init() below?
+}
+
+i18next
+    .use(ICU)
+    .init({
+        debug: true,
+        initImmediate: false,
+        fallbackLng: 'en',
+        lng: getLocale(supportedLocales),
+        ns: ['shared', 'options'],
+        defaultNS: 'shared',
+        resources: buildLocalesList()
+    })
+
```

Comment:
Just a nit, but if we're always getting the `title` attribute of these json objects, can we hide that away so to make things more concise? Like:

```js
const i18T = (token) => i18next.t(`${token}.title`)

module.exports = i18T
```

Then we can use it like `t('shared:learnMore')` and omit the `.title`.

Reply:
I started down this path, but then realized there are a few other useful functions that we'd have to wrap, like `i18n.exists`.  Making this change will make key management a bit of a muddle, or require us to proxy other calls as well.  Feels like a bit of a rabbit hole TBH, as we should also then make this change in the extension (and become a pattern across our codebase).

Reply:
I'm going to merge as is for now, given it would touch a lot of code and probably should also be done for the dashboard.  Its a good idea, but maybe better done as a followup.